app/assets/javascripts/assignments.js

```diff
@@ -25,7 +25,13 @@ $(document).ready(function() {
                             klass = 'btn-success';
                             break;
                     }
-                    $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" >' + actions_map[mid].action + '</a>')
+                    if ( actions_map[mid].action == 'Attempt' ){
```

Comment:
I'm afraid that if I touched other lines, new complains will come and finally I need to fix the whole file ><